dev-tools/packer/Makefile

```diff
@@ -3,11 +3,11 @@ BUILDID?=$(DATE)
 
 
 .PHONY: all
-all: packetbeat/deb packetbeat/rpm packetbeat/darwin packetbeat/win packetbeat/bin \
+all: packetbeat/deb packetbeat/rpm packetbeat/darwin packetbeat/win32 packetbeat/bin \
 	topbeat/deb topbeat/rpm topbeat/darwin topbeat/win topbeat/bin \
-	filebeat/deb filebeat/rpm filebeat/darwin filebeat/win filebeat/bin \
-	metricbeat/deb metricbeat/rpm metricbeat/darwin metricbeat/win metricbeat/bin \
-	winlogbeat/win \
+	filebeat/deb filebeat/rpm filebeat/darwin filebeat/win32 filebeat/bin \
+	metricbeat/deb metricbeat/rpm metricbeat/darwin metricbeat/win32 metricbeat/bin \
+	winlogbeat/win32 \
```

Comment:
Ah, now I see you only added it for Topbeat? Shouldn't we rather do it for all beats?